app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,37 @@
+<main class="index_container">
+  <h2>Sign up</h2>
+  <div class="form_style">
+    <%= form_for(resource, as: resource_name,  data: {turbo: false}, url: registration_path(resource_name)) do |f| %>
+      <%= render "devise/shared/error_messages", resource: resource %>
+
+      <div class="field">
+        <%= f.label :name, "Username" %><br />
+        <%= f.text_field :name, autofocus: true, autocomplete: "name", placeholder: "Email", class:"input_style" %>
```

Comment:
- [ ] For the `sign up` page, the placholder text for the first input box should be `username`, kindly fix this. 
![image](https://user-images.githubusercontent.com/81932478/216369553-7a195a35-74a9-4ea0-9586-39479bc29ef7.png)